dotnetAvaloniaFrameBuffer/Dockerfile

```diff
@@ -31,24 +37,59 @@ RUN dotnet restore && \
 dotnet publish -c Release -r linux-${IMAGE_ARCH} --no-self-contained
 # BUILD ------------------------------------------------------------------------
 
+# DOTNET -----------------------------------------------------------------------
+FROM --platform=linux/${IMAGE_ARCH} \
+    torizon/dotnet:${DOTNET_BASE_VERSION} AS Dotnet
 
 
 # DEPLOY -----------------------------------------------------------------------
 FROM --platform=linux/${IMAGE_ARCH} \
-    torizon/dotnet-uno:${BASE_VERSION} AS Deploy
+    torizon/wayland-base${GPU}:${BASE_VERSION} AS Deploy
 
 ARG IMAGE_ARCH
+ARG GPU
 ARG APP_EXECUTABLE
 ENV APP_EXECUTABLE ${APP_EXECUTABLE}
 
+ENV DOTNET_ROOT=/dotnet
+ENV PATH=$PATH:/dotnet
+
+COPY --from=Dotnet /dotnet /dotnet
+
+# for vivante GPU we need some "special" sauce
+RUN apt-get -q -y update && \
+        if [ "${GPU}" = "-vivante" ]; then \
+            apt-get -q -y install \
+            imx-gpu-viv-wayland-dev \
+        ; else \
+            apt-get -q -y install \
+            libgl1 \
+        ; fi \
+    && \
+    apt-get clean && apt-get autoremo
```

Comment:
Missing the end part here:
`    apt-get clean && apt-get autoremove && \
    rm -rf /var/lib/apt/lists/*`